### PR TITLE
Issue 3012: Document fix: segmentstore_service.md

### LINKED ISSUE #3012: Document review segment-store-service.md
**Problem description**
1. Checking formats
2. Checking hyperlinks

**Problem location**
segment-store-service.md

### PR DESCRIPTION
**Pull request by Aparnarr**
**Change log description**  
1. Formats Fixed
2. Hyperlinks Fixed

**Purpose of the change**  
 Fixes #3012

### CODE REVIEW

File: documentation/src/docs/segment-store-service.md [hunk 1]
```diff
@@ -130,17 +132,18 @@ The _Durable Log_ is the central component that handles all Log Operations. All
 1. All received operations are added to an _Operation Queue_ (the caller receives a Future which will be completed when the operation is durably persisted).
 2. The _Operation Processor_ picks all operations currently available in the queue (if the queue is empty, it will wait until at least one Operation is added).
 3. The _Operation Processor_ runs as a continuous loop (in a background thread), and has four main stages.
```
Comment (Aparnarr) [inline documentation/src/docs/segment-store-service.md]: It is mention in  3rd point that  `The _Operation Processor_ runs as a continuous loop (in a background thread), and has four main stages.` but only 3 points are mentioned below: please check

File: documentation/src/docs/segment-store-service.md [hunk 2]
```diff
@@ -18,75 +18,76 @@ The Pravega Segment Store Service can provide the following guarantees:
...
 
-More details about each component described above can be found in the [Components](#Components) section.
+More details about each component described above can be found in the [Components](#components) section.
 
 ## System Diagram
 
 ![System Diagram](img/Segment.Store.Components.png)
 
-In the above diagram, the major components of the Segment Store are shown. But for simplicity, only one Segment Container is depicted. All Container components and major links between them (how they interact with each other) are shown. The _Container Metadata_ component is not shown, because every other component communicates with it in one form or another, and adding it would only clutter the diagram.
+In the above diagram, the major components of the Segment Store are shown. But for simplicity, only one Segment Container is depicted. All Container components and major links between them (how they interact with each other) are shown. The _Container Metadata_ component is not shown, because every other component communicates with it in, one form or another, and adding it would only clutter the diagram.
```
Comment (andreipaduroiu) [inline documentation/src/docs/segment-store-service.md]: remove the added comma between `in` and `one`

File: documentation/src/docs/segment-store-service.md (hunk 1 above)
Comment (andreipaduroiu) [reply documentation/src/docs/segment-store-service.md]: then remove the number and say `executes the following steps`

Comment (Aparnarr) [reply documentation/src/docs/segment-store-service.md]: Accepted suggestion

File: documentation/src/docs/segment-store-service.md (hunk 2 above)
Comment (Aparnarr) [reply documentation/src/docs/segment-store-service.md]: Accepted Suggestion

**Pull request merged**